lib/ts/unit-tests/test_MemArena.cc

```diff
@@ -92,6 +94,20 @@ TEST_CASE("MemArena freeze and thaw", "[libts][MemArena]")
 
 TEST_CASE("MemArena helper", "[libts][MemArena]")
 {
+  struct Thing {
+    int ten{10};
+    std::string name{"name"};
+
+    Thing() {}
+    Thing(int x) : ten(x) {}
+
+    Thing(std::string const &s) : name(s) {}
```

Comment:
Needed here, or add a test case below?

Reply:
I think that ends up getting used in the `PrivateThing` tests.

Reply:
OK let's remove and add when we actually use the code.

Reply:
Looks good.